business_requirement_renew/models/business_requirement_renew.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models, api, fields, _
+
+
+class BusinessRequirementRenew(models.Model):
+    _inherit = 'business.requirement'
+
+    source_id = fields.Many2one(
+        'business.requirement',
+        string="Original Bus.Req",
+        readonly=True)
+    copy_from_id = fields.Many2one(
+        'business.requirement', string="Renewed From")
+    version = fields.Integer(string='Version', default=0)
+
+    @api.multi
+    def _compute_br_children_count(self):
```

Comment:
Better not use the term "child" or "children" because the function exist in BR (parent/child relationship). This is confusing.
Better use "_compute_related_br_count" (please apply the rule in the whole module)